modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryPerformanceTest.java

```diff
@@ -151,7 +151,9 @@ public void testGetObjectEntries() throws Exception {
 	}
 
 	@Test
-	public void testImportAndDeleteByRestAPI() throws Throwable {
+	public void testImportAndDeleteObjectEntriesByHttpInvoker()
```

Comment:
@slnn @tinatian 

> 3.) This is a good name: `testGetObjectEntries`
> because I can match it to `_objectEntryLocalService.getObjectEntries`
> But this is a very bad name `testImportAndDeleteObjectEntriesByRestAPI`

I wonder if we should call this method `testHttpPostAndDeleteObjectEntries` so that it matches to `_invokeHttp(..., HttpMethod.POST)`, and `_invokeHttp(..., HttpMethod.DELETE)`.

Reply:
OK